packages/node/src/index.ts

```diff
@@ -413,6 +413,11 @@ export const build: BuildV3 = async ({
         )} (must be one of: ${JSON.stringify(ALLOWED_RUNTIMES)})`
       );
     }
+    if (staticConfig.runtime === 'nodejs') {
```

Comment:
**suggestion (non-blocking):** Just in case we forget to come back to remove this, do you want to add a comment?

```suggestion
    // TODO: Remove this after we learn how often `nodejs` is
    // explicitly set as a `runtime` value
    if (staticConfig.runtime === 'nodejs') {
```

Reply:
I actually think we should turn it into a warning or error depending on the results we find